flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/EnvironmentSettings.java

```diff
@@ -149,19 +149,26 @@ public boolean isStreamingMode() {
 	 * A builder for {@link EnvironmentSettings}.
 	 */
 	public static class Builder {
-		private String plannerClass = null;
-		private String executorClass = null;
+		private static final String OLD_PLANNER_FACTORY = "org.apache.flink.table.planner.StreamPlannerFactory";
+		private static final String OLD_EXECUTOR_FACTORY = "org.apache.flink.table.executor.StreamExecutorFactory";
+		private static final String BLINK_PLANNER_FACTORY = "org.apache.flink.table.planner.delegation.BlinkPlannerFactory";
+		private static final String BLINK_EXECUTOR_FACTORY = "org.apache.flink.table.planner.delegation.BlinkExecutorFactory";
+
+		private String plannerClass = OLD_PLANNER_FACTORY;
+		private String executorClass = OLD_EXECUTOR_FACTORY;
 		private String builtInCatalogName = "default_catalog";
 		private String builtInDatabaseName = "default_database";
 		private boolean isStreamingMode = true;
 
 		/**
 		 * Sets the old Flink planner as the required module. By default, {@link #useAnyPlanner()} is
```

Comment:
Please update the javadoc: `By default, {@link #useAnyPlanner()} is` -> `By default, {@link #useOldPlanner()} is`. Also in the remaining two methods.

Reply:
Good catch!